backend/user-service/src/main/java/com/seederuserservice/controller/UserController.java

```diff
@@ -0,0 +1,45 @@
+package com.seederuserservice.controller;
+
+import com.seederuserservice.dto.request.PatchRequest;
+import com.seederuserservice.dto.request.PostUserRequest;
+import com.seederuserservice.dto.response.GetUserResponse;
+import com.seederuserservice.service.IUserService;
+import java.util.UUID;
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RequestMapping("/api/v1/users")
+@RestController
+public class UserController {
+
+  private final IUserService userService;
+
+  @Autowired
+  public UserController(IUserService iUserService) {
+    this.userService = iUserService;
```

Comment:
- why the name IUserService?
- remve the I

Reply:
Its a naming convention where I stands for interface